validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -718,14 +718,14 @@ private void loadSiteMetadata() {
     } catch (FileNotFoundException e) {
       return;
     } catch (Exception e) {
-      throw new RuntimeException("While validating " + SITE_METADATA_JSON, e);
+      throw new RuntimeException("While validating " + SITE_DEFAULTS_JSON, e);
     }
 
     try {
-      System.err.printf("Loading " + SITE_METADATA_JSON + "\n");
+      System.err.printf("Loading " + SITE_DEFAULTS_JSON + "\n");
       this.siteMetadata = OBJECT_MAPPER.readValue(siteMetadataFile, Metadata.class);
```

Comment:
so -- if we're changing the schema then we need to change the class it's using as well... So rather than loading this into Metadata.class it should load it into SiteMetadata.class